### DISCUSSION TITLE
Can I use my ForwardEmail.Net account to send emails using nodeemailer?

### DISCUSSION THREAD

**Discussion by breck7** [forwardemail/forwardemail.net#183] [Q&A] [answered]
This is a dumb question but I can't figure it out.

I found out about ForwardEmail.net on this page: https://nodemailer.com/about/.

So I signed up for a year and moved my email forwarding to it away from Google Domain's basic catch all. That works.

But now I want to use `nodeemailer` to send emails, something like the below. Is this possible?

```
const { user, pass, host, port, secure } = this.emailConfig
    let transporter = nodemailer.createTransport({
      host: "forwardemail.net",
      port: 25,
      secure: true,
      auth: {
        user,
        pass
      }
    })
```

**Comment by titanism** [comment]
We are working on and launching SMTP soon - which would make this possible.

**Reply by titanism** [reply]
In the interim, you could use a service such as Postmark App or Twilio Sendgrid.

**Reply by breck7** [reply]
Got it! Thank you so much for the fast response @titanism ! I was able to get it working with Twilio Sendgrid.

**Comment by titanism** [comment, marked as answer]
Hey just following up here cc @breck7  we support outbound SMTP now 😄 

https://forwardemail.net/en/docs/how-to-javascript-contact-forms-node-js

https://forwardemail.net/en/guides/send-email-with-custom-domain-smtp

**Reply by hkdobrev** [reply]
Do you plan to offer simple pricing for sending more than 9k transactional emails per month and without a daily limit?

**Reply by titanism** [reply]
@hkdobrev $1/mo right now per 1000 messages, billed monthly (divided average amongst 30 days), we can easily add more if you need just message us over Element/Matrix chat or file a support ticket/email
